### DISCUSSION TITLE
Roll up window title bar

### DISCUSSION THREAD

**Discussion by nuntavits** [WindowTop/WindowTop-App#394] [Ideas]
Hi,

Thank you for creating this wonderful app. The only missing feature that I'd like which is roll up, please see image attached. I'm working with an application that has a lot of small windows, and being able to roll the window up, I can easily access what's underneath.
![ss_20241105164451](https://github.com/user-attachments/assets/4ae5e96f-82cb-4303-b7b1-117c4cc06c80)

**Comment by gileli121** [comment]
Hello, did you try the Anchors feature?  
More about it here:  
https://windowtop.info/features/anchors/  

It provides a similar function to what you’re looking for—maintaining easy access to a window while minimizing the space it takes on the screen. Anchors achieve this more effectively by using floating icons that occupy far less space.  

Not only that, but these floating icons automatically move away from text or images to avoid covering important content (like something you want to read). The roll-window concept doesn't offer this; if there is text beneath a rolled window that you want to view, you need to move the window manually. With Anchors, a smart detection algorithm handles this automatically.  

One minor downside compared to the win-roll concept is that there isn't an option to enable an anchor for a specific window. This feature has already been requested here: #183 . The idea is to add a new "Anchor" button to the toolbar, which, when enabled, would create an anchor only for that window. Currently, the app auto-generates anchors for any window that is completely behind other windows.  

If you like, there is an option to limit the maximum number of generated anchors (refer to the link above). However, I recommend getting used to minimizing unused anchors with a right-click of the mouse.

Please let me know your thoughts on this and whether it seems like a good solution.  
I would be happy to hear any suggestions or feedback.  

Thanks!
